modules/tinymce/src/core/main/ts/paste/Clipboard.ts

```diff
@@ -269,15 +268,14 @@ const registerEventHandlers = (editor: Editor, pasteBin: PasteBin, pasteFormat:
       e.preventDefault();
       insertClipboardContent(editor, clipboardContent, clipboardContent['text/plain'], plainTextMode, true);
     } else {
+      e.preventDefault();
```

Comment:
I'm surprised thst this code path is executing at all this is a fallback if the clipboard ap exists or doesn't have html or plaintext mime types.
Also preventing the paste event will make this code not execute at all since why it has a delay is that we late the paste operation fill the paste bin then we extract what it pastes from the bin if you prevent that default action from happening you should have a empty bin always so basically this code with this change should not work at all. Are we sure that this works, under what condition is this code path executed?
Also powerpaste is async on regular html so does this bug happen on that, so not sure this is the solution we are looking for.

Reply:
The comment on the next line explains what's going on (`We can't extract...`) and why this change can never work. The comment is as clear as it can be without doing a deep-dive into browser events; this is just one of those things that has to be learnt when it comes up.

The original ticket included a video that should've made it clear this was the wrong place to tackle the problem.